test/expect/TestJit.test_cpp_cuda.expect

```diff
@@ -135,35 +153,53 @@ graph(%0 : Float(*)
   %3 : int = prim::Constant[value=1]()
   %4 : Float(*) = aten::add(%2, %1, %3)
   %5 : int = prim::Constant[value=1]()
+  %26 : int[] = aten::size(%4)
```

Comment:
nit: It might be good to add @ngimel's example from #11736 into test/test_jit.py to have a non-expect-test that checks the derivatives of a broadcasting op are okay.

Reply:
Good point, I forgot to do that, even though I tested this solution on that snippet. Will do that in my followup PR that cleans up the DCE situation.